src/controlplane-metadata/src/spg/k8/spec.rs

```diff
@@ -56,8 +56,6 @@ pub struct SpuTemplate {
     #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
crate version number need to be updated and all effected crates should use new version otherwise this will cause build issue

Reply:
bumped crate versions